inventory_management_system_api/repositories/catalogue_category.py

```diff
@@ -211,7 +220,7 @@ def _is_duplicate_catalogue_category(
         parent_id: Optional[str],
         code: str,
         catalogue_category_id: CustomObjectId = None,
```

Comment:
Probably worth updating this one too
```suggestion
        catalogue_category_id: Optional[CustomObjectId] = None,
```

Reply:
Good spot, I have just done a global search and found some more as well.